### DISCUSSION TITLE
Help to decode signal pattern

### DISCUSSION THREAD

**Discussion by habub** [merbanan/rtl_433#2486] [General]
@zuckschwerdt appreciate if you can take a look at these signal files. I can't find any pattern in the data payload. Maybe you can assist?
Attached cu8 sample files....
Thanks

[sample_files.zip](https://github.com/merbanan/rtl_433/files/11286491/sample_files.zip)

**Comment by zuckschwerdt** [comment]
Looks like PWM, e.g. `-X 'n=name,m=OOK_PWM,s=120,l=220,r=250'`
The repeats seem to be the same but each transmission is different. Maybe a rolling code.
